WordPress/src/main/res/values-small/dimens.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="avatar_sz_login">64dp</dimen>
+    <dimen name="avatar_sz_login_epilogue">38dp</dimen>
+    <dimen name="margin_extra_extra_large">24dp</dimen>
+    <dimen name="margin_extra_large">8dp</dimen>
+    <dimen name="magic_link_sent_illustration_sz">72dp</dimen>
+    <dimen name="margin_medium_large">4dp</dimen>
```

Comment:
Altering `margin_medium_large` has minimal effect AFAICT so, what do you think about not differentiating it? It will keep the scope down as well.